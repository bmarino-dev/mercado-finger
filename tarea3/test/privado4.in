crearHistorial 01/01/2024
crearPromocion 01/12/2023 01/01/2024 10 5
agregarPromocionHistorial
crearPromocion 02/01/2024 15/02/2024 10 6
agregarPromocionHistorial
crearFecha 01/01/2024
crearProducto 1 Producto1 Autor1 Apellido1
agregarProductoAPromocionHistorial 5
crearFecha 02/02/2024
crearProducto 2 Producto2 Autor2 Apellido2
agregarProductoAPromocionHistorial 6
imprimirPromocionesFinalizadosHistorial
imprimirPromocionesActivasHistorial
imprimirPromocionesFuturasHistorial
avanzarAFechaHistorial 05/01/2024
imprimirPromocionesFinalizadosHistorial
imprimirPromocionesActivasHistorial
imprimirPromocionesFuturasHistorial
crearPromocion 01/12/2023 15/12/2023 10 1
agregarAPromocion 2 6 5
esCompatiblePromocionHistorial
liberarPromocion
liberarHistorial
Fin
