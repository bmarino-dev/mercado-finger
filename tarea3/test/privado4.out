Bienvenido al programa principal de MercadoFinger. Por favor ingrese su comando:
1> Historial creado con exito.
2> Promocion creada con exito.
3> La promocion fue agregada al historial con exito.
4> Promocion creada con exito.
5> La promocion fue agregada al historial con exito.
6> Fecha creada en forma exitosa.
7> Producto creado en forma exitosa.
8> El producto fue agregado a la promocion de id 5.
9> Fecha creada en forma exitosa.
10> Producto creado en forma exitosa.
11> El producto fue agregado a la promocion de id 6.
12> Promociones finalizadas:
13> Promociones activas:
Promocion #5 del 1/12/2023 al 1/1/2024
Productos: 1 
14> Promociones futuras:
Promocion #6 del 2/1/2024 al 15/2/2024
Productos: 2 
15> Se avanzo la fecha de la historial a 5/1/2024
16> Promociones finalizadas:
Promocion #5 del 1/12/2023 al 1/1/2024
Productos: 1 
17> Promociones activas:
Promocion #6 del 2/1/2024 al 15/2/2024
Productos: 2 
18> Promociones futuras:
19> Promocion creada con exito.
20> Productos agregados a la promocion de forma exitosa.
21> La promocion es compatible con las demas promociones de la historial.
22> Promocion liberada con exito.
23> Historial liberada con exito.
24> Fin.
